i18n/hi/fotema.ftl

```diff
@@ -0,0 +1,219 @@
+
+## Terms
+# See https://projectfluent.org/fluent/guide/terms.html
+
+-app-name = Fotema
+
+## Main Navigation Pages
+
+# Title for library page, which contains the "all", "months", and "years" pages.
+library-page = लाइब्रेरी
+
+# Title for years album.
+years-album = वर्ष
+
+# Title for months album.
+months-album = महीना
+
+# Title for all photos/videos album.
+all-album = दिन
+
+# Title for video album.
+videos-album = वीडियो
+
+# Title for album of selfies.
+selfies-album = सेल्फी
+
+# Title for album of iOS live photos and Android motion photos.
+animated-album = सजीवित
+
+# Title for album showing all folders.
+folders-album = फोल्डर
+
+# Title for album showing contents of one folder.
+folder-album = फोल्डर
+
+## Thumbnail decorations
+
+# Label on month album thumbnails.
+# Variables:
+#   $month - month number (1 through 12).
+#   $year - year e.g., 2024
+month-thumbnail-label = { $month ->
+   [1] जनवरी {$year}
+   [2] फरवरी {$year}
+   [3] मार्च {$year}
+   [4] अप्रैल {$year}
+   [5] मई {$year}
+   [6] जून {$year}
+   [7] जुलाई {$year}
+   [8] अगस्त {$year}
+   [9] सितम्बर {$year}
+   [10] अक्टूबर {$year}
+   [11] नवम्बर {$year}
+   [12] दिसम्बर {$year}
+  *[अन्य] {$year}
+  }
+}
```

Comment:
This is an extra closing brace which also exists in `en-US/fotema.ftl`  (which I'll fix shortly). When I remove the extra brace each `fotema.ftl` file and fix the naming of the default case, then month labels translate correctly.

Reply:
I've pushed a fix to `main` to remove the extra closing brace in `i18n/en-US/fotema.ftl`.

Reply:
Fixed in translated file.